blackjack/core.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+import enum
+import random
+from typing import List, Optional
+
+
+class Win(enum.Enum):
+    Push = enum.auto()
+    Player_bj = enum.auto()
+    House_bj = enum.auto()
+    Player_bust = enum.auto()
+    House_bust = enum.auto()
+    Player_win = enum.auto()
+    House_win = enum.auto()
```

Comment:
Bad casing here. Use `CamelCase` here, e.g. `PlayerBlackjack`, `HouseBust`, etc.

Also, the name `Win` is not good since the cases don't all represent "win" scenarios.